<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempoWidget</class>
 <widget class="QWidget" name="TempoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="tempoGroup">
     <property name="title">
      <string>Tempo</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="stepButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources/Resources.qrc">
          <normaloff>:/Step.svg</normaloff>:/Step.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playPauseButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources/Resources.qrc">
          <normaloff>:/Play.svg</normaloff>:/Play.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="bpmSpin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> bpm</string>
        </property>
        <property name="minimum">
         <number>30</number>
        </property>
        <property name="maximum">
         <number>250</number>
        </property>
        <property name="value">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources/Resources.qrc">
          <normaloff>:/Reset.svg</normaloff>:/Reset.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
